spec/paperclip/storage/s3_spec.rb

```diff
@@ -14,6 +14,21 @@ def aws2_add_region
     defined?(::Aws) ? { s3_region: 'us-east-1' } : {}
   end
 
+  context 'multithreaded initialization' do
+    before do
+    end
+    it "should not fail on missing constants" do
+      10.times.map do |i|
+        Thread.new do
+          ActiveRecord::Base.connection_pool.with_connection do |_|
+            rebuild_model (aws2_add_region).merge storage: :s3
+            Dummy.new.avatar
+          end
+        end
+      end.map{|t| t.join}
```

Comment:
Avoid multi-line chains of blocks.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:join` as an argument to `map` instead of a block.